lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
```

Comment:
As noted earlier, this whole partial is a move operation from styles in the _discussion.scss partial that was cleaned up for other parts of this PR. There are no changes otherwise within this section.